apps/theming/lib/ThemingDefaults.php

```diff
@@ -420,6 +420,18 @@ public function replaceImagePath($app, $image) {
 		}
 
 		if ($route) {
+			$userId = '';
+			try {
+				$userId = $this->userSession->getUser()->getUID();
+			} catch (\Throwable $e) {
+			} catch (\Exception $e) {
+			}
```

Comment:
We cannot inject the userId here because this class is loaded too early, but something like this might be a bit nicer to read:

```suggestion
			$userId = $this->userSession->getUser() ? $this->userSession->getUser()->getUID() : '';
```

Reply:
thanks